frontend/pages/playscreen.vue

```diff
@@ -0,0 +1,465 @@
+  const maxDashOffset = 2 * Math.PI * timerRadius.value; // Umfang des Kreises
+  return (remainingTime.value / totalTime.value) * maxDashOffset; // Rückwärtslaufender Balken
+});
+
+// Anzeige des Game Timers als numerische Zeitangabe
+const gameTimerDisplay = computed(() => remainingGameTime.value);
+
+// Funktion zum Mischen des Arrays
+const shuffleArray = (array) => {
+  for (let i = array.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [array[i], array[j]] = [array[j], array[i]];
+  }
+  return array;
+};
+
+// Timer-Logik für das Trisight Mode Spiel
+const startGameTimer = () => {
+  clearInterval(gameTimer.value);
+  gameTimer.value = setInterval(() => {
+    if (remainingGameTime.value > 0) {
+      remainingGameTime.value -= 1;
+    } else {
+      clearInterval(gameTimer.value);
+      endGame(); // Beendet das Spiel, wenn die Zeit abgelaufen ist
+    }
+  }, 1000); // Der Timer reduziert sich jede Sekunde
+};
+
+// Funktion zum Abrufen der Daten basierend auf dem Schwierigkeitsgrad
+const fetchData = async () => {
+  try {
+    let endpoint;
+    if (isTrisightMode.value) {
+      switch (difficulty.value) {
+        case 'medium':
+          endpoint = "http://localhost:8000/api/test_configs/3";
+          break;
+        case 'hard':
+          endpoint = "http://localhost:8000/api/test_configs/4";
+          break;
+        default:
+          endpoint = "http://localhost:8000/api/test_configs/1"; // Leicht (Standard)
+      }
+    } else {
+      endpoint = "http://localhost:8000/api/test_configs/2"; // Endpunkt für den Standardmodus
+    }
+    const response = await $fetch(endpoint);
+    return response;
+  } catch (error) {
+    console.error("Fehler beim Abrufen der Konfigurationen:", error);
+    return null;
+  }
+};
+
+const data = await fetchData();
+console.log(data);
+
+const initializeGame = () => {
+  if (data && data.item_configs.length > 0) {
+    randomizedItems.value = isTrisightMode.value
+      ? shuffleArray([...data.item_configs]) // Zufällig gemischte Kopie der item_configs für Trisight Mode
+      : [...data.item_configs]; // Kopie der item_configs für den Standardmodus
+    currentItem.value = randomizedItems.value[0]; // Setze das erste Item als Start
+    totalTime.value = currentItem.value.time_visible_ms;
+    remainingTime.value = totalTime.value;
+  }
+};
+
+const startGame = () => {
+  isGameStarted.value = true;
+  if (isTrisightMode.value) {
+    startTimer();
+    startGameTimer(); // Starte den universellen Game Timer
+  }
+};
+
+const startTimer = () => {
+  clearInterval(timer.value);
+  timer.value = setInterval(() => {
+    if (remainingTime.value > 0) {
+      remainingTime.value -= 10; // Aktualisierungsrate auf 10 ms gesetzt
+    } else {
+      clearInterval(timer.value);
+      checkAnswer();
+    }
+  }, 10); // Timer-Intervall auf 10 ms gesetzt
+};
+
+const handleKeyPress = (event) => {
+  if (!isGameStarted.value) {
```

Comment:
I started with space and it scrolled down the page. Maybe we need to supress the default behaviour for this as well